third_party/move/move-prover/tests/sources/functional/duplicate_function_declarations.move

```diff
@@ -0,0 +1,15 @@
+module 0x42::DuplicateFunction {
+    struct R0 { x: u8 }
+
+    spec R0 {
+        fun double(x: u8): u8 {
```

Comment:
Do you think this duplicated function is necessary to reproduce the problem? Otherwise, can we remove it to avoid the error above?

Reply:
The duplicated function is neccessary to trigger the error.